content/en/blog/_posts/2022-4-26-kubernetes-release-1.24.md

```diff
@@ -0,0 +1,149 @@
+---
+layout: blog
+title: "Kubernetes 1.24: Stargazer"
+date: 2022-05-03
+slug: kubernetes-1-24-release-announcement
+---
+
+# Kubernetes 1.24
+
+**Authors**: [Kubernetes 1.24 Release Team](https://github.com/kubernetes/sig-release/blob/master/releases/release-1.24/release-team.md)
+
+We are excited to announce the release of Kubernetes 1.24, the first release of 2022!
+
+This release consists of 46 enhancements: fifteen enhancements have graduated to stable, fifteen enhancements are moving to beta, and thirteen enhancements are entering alpha. Also, two features have been deprecated, and the [dockershim has been removed](https://kubernetes.io/blog/2022/02/17/dockershim-faq/).     
+
+## Major Themes
+
+### Dockershim Removed from kubelet
+After an initial deprecation in v1.20, the dockershim has been removed from the kubelet in favor of runtimes that comply with the Container Runtime Interface (CRI) designed for Kubernetes. From v1.24 and up, if you are currently relying on Docker Engine as your container runtime, you will need to either use one of the other supported runtimes (such as containerd or CRI-O) or use cri-dockerd. For more information about ensuring your cluster is ready for this removal, please see [this guide](/blog/2022/03/31/ready-for-dockershim-removal/). 
+  
+### CNI Version-Related Breaking Change
+Before you upgrade to Kubernetes 1.24, if you use a [Container Network Interface (CNI)](https://github.com/containernetworking/cni) network plugin that complies with CNI versions earlier than v1.0.0 and you plan to use any of the following container runtimes, you must specify the CNI version as described in [Declaring the CNI version of your plugin](TBD). Otherwise, your network will break. 
+
+This issue applies to the following container runtimes:
+* containerd v1.6.0 and later
+* CRI-O 1.24 and later
+
+### Beta APIs Off by Default 
+[New beta APIs will not be enabled in clusters by default](https://github.com/kubernetes/enhancements/issues/3136). Existing beta APIs and new versions of existing beta APIs, will continue to be enabled by default.
+  
+### OpenAPI v3
+Kubernetes 1.24 offers beta support for publishing its APIs as [OpenAPI v3](https://github.com/kubernetes/enhancements/issues/2896). 
+
+### Signing Release Artifacts
+[Release artifacts are signed](https://github.com/kubernetes/enhancements/issues/3031). 
```

Comment:
Could we have:
```suggestion
Release artifacts are [signed](https://github.com/kubernetes/enhancements/issues/3031)
and there is experimental support for [verifying image signatures](/docs/tasks/administer-cluster/verify-signed-images/).
```
?

See https://deploy-preview-31225--kubernetes-io-main-staging.netlify.app/docs/tasks/administer-cluster/verify-signed-images/ for a preview of that linked task page.

Are _all_ artifacts signed, or just some?